templates/getting_started_python/.actor/Dockerfile

```diff
@@ -3,9 +3,10 @@
 # You can also use any other image from Docker Hub.
 FROM apify/actor-python:3.9
```

Comment:
Any particular reason for 3.9?

Reply:
Not really. I was planning to update it to `3.11`, but we don't have `3.11` stable yet, I would increase it after we release the Python SDK and the accompanying Docker images with it.